google/cloud/aiplatform/docker_utils/build.py

```diff
@@ -191,23 +185,18 @@ def _prepare_entrypoint(package: Package, python_command: str = "python") -> str
 
 
 def _prepare_package_entry(package: Package) -> str:
-    """Returns the Dockerfile entries required to append at the end before entrypoint.
-
-    Including:
-    - copy the parent directory of the main executable into a docker container.
-    - inject an entrypoint that executes a script or python module inside that
-      directory.
+    """Returns the Dockerfile entries required to copy the packages to images.
 
     Args:
         package (Package):
             Required. The main application copied to and run in the container.
 
     Returns:
-        The generated package related command used in Dockerfile.
+        The generated package related copy command used in Dockerfile.
     """
     copy_code = _generate_copy_command(
         ".",  # Dockefile context location has been changed to host_workdir
-        Path(package.package_path).name,
```

Comment:
What did this copy previously? This was called with our `google.cloud.aiplatform.prediction.model_server` package?

Reply:
This copied all files in the users' specified directory to a directory with the same name under the working directory of images.